charts/plugin-br-pix-indirect-btg/templates/reconciliation/deployment.yaml

```
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "reconciliation.fullname" . }}
+  labels:
+    {{- include "reconciliation.labels" (dict "context" . "name" .Values.reconciliation.name ) | nindent 4 }} 
+spec:
+  revisionHistoryLimit: {{ .Values.reconciliation.revisionHistoryLimit | default 10 }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)